untangle/ngfw_src
# ngfw-14374: remove community languages

This PR covers removal community language reference from NGFW , where as removal of community" information hosted under a separate server cannot be covered in this PR


Acceptance Criteria 
Remove community language source from language selection in Config->System->Language tab of NGFW

Built "Build i18n template for localization" task locally. 

Official languages got built successfully and ther zips were created. PFA
![Screenshot from 2024-02-29 19-02-45](https://github.com/untangle/ngfw_src/assets/154513962/2fdead22-7b20-4ccc-98ef-280fc10fa2fe)


Testing steps:
Upgrade NGFW with these change and restart ngfw
community language source from language selection in Config->System->Language tab of NGFW should be removed
PFA

![Screenshot from 2024-02-29 19-35-35](https://github.com/untangle/ngfw_src/assets/154513962/2595764f-64d0-4b8c-a7e5-e5f905b08030)

Note if set up has community language set up, after upgrade and restart:
Source, Language will be set to default which is Official and English.
